### ISSUE TITLE
When will you release pre-trained language models based on sigmoid attention

### ISSUE THREAD

**Issue by guxm2021** [apple/ml-sigmoid-attention#3]
Thank you for your impressive work, congratulations! I am wondering when you will release pre-trained language models based on sigmoid attention, at least a small demo models? I am looking forward to use your models.

**Comment by jramapuram** [comment]
Hi thanks for the interest! We don't currently have any plans to release the models unfortunately.

**Comment by WeersProductions** [comment]
We did release them, just now 😄 
@guxm2021 see the notebook on how to run inference using AXLearn: https://github.com/apple/ml-sigmoid-attention/blob/main/pretrained/axlearn_load_pretrained.ipynb

The intermediate checkpoints are also released.

**Comment by jramapuram** [comment]
Let us know if you have any issues!
